_posts/2016/2016-06-14-gradle-jenkins-plugin.md

```diff
@@ -1,19 +1,20 @@
 ---
 layout: post
 title: "How to empower your Jenkins with Gradle"
-date: "2016-05-31 10:59:11"
+date: "2016-06-14 10:59:11"
+image: blog-head/gradle-jenkins-tools.jpeg
```

Comment:
blog-head/gradle-jenkins-tools.jpeg
>
blog-header/gradle-jenkins-tools.jpg

please use the image I sent you via HipChat